pyhole/core/plugin.py

```diff
@@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
             __import__("pyhole.plugins", globals(), locals(), [plugin_name])
         except Exception, exc:
             LOG.error(exc)
+            raise
```

Comment:
To give an example, if the bot is connected to multiple IRC or Slack networks at any given time, a syntax error in a Slack plugin on network "A" should not crash the core process and halt the bot on networks "B", "C", "D", etc...